src/components/footer/grape-input/GrapeInput.js

```diff
@@ -391,6 +391,7 @@ class GrapeInput extends PureComponent {
       onHideBrowser,
       onRequestAutocompleteServicesStats,
       goTo,
+      channel,
```

Comment:
Why is passing in the channel and [this part in grape browser](https://github.com/ubergrape/grape-browser/pull/103)  necessary since we already have `hideBrowser`?

Reply:
@nikgraf to detect if channel changed - https://github.com/ubergrape/grape-browser/pull/103/files#diff-16358613cf043170211d8d22566d609fR62

Reply:
Discussed in the office with @AleshaOleg that this is needed to restore the browser when switching back to a channel with an open mention for example. Will add a comment in the Grape Browser